imap_processing/tests/spice/test_time.py

```diff
@@ -1 +1,30 @@
 """Tests coverage for imap_processing/spice/time.py"""
+
+import numpy as np
+import pytest
+import spiceypy as spice
+
+from imap_processing.spice import IMAP_SC_ID
+from imap_processing.spice.time import TICK_DURATION, _sct2e_wrapper, met_to_j2000ns
+
+
+def test_met_to_j2000ns(furnish_time_kernels):
+    """Test coverage for met_to_j2000ns function."""
+    utc = "2026-01-01T00:00:00.125"
+    et = spice.str2et(utc)
+    sclk_str = spice.sce2s(IMAP_SC_ID, et)
+    seconds, ticks = sclk_str.split("/")[1].split(":")
+    met = float(seconds) + float(ticks) * TICK_DURATION
+    j2000ns = met_to_j2000ns(met)
+    assert j2000ns.dtype == np.int64
+    assert j2000ns == et * 1e9
+
+
+@pytest.mark.parametrize("sclk_ticks", [0.0, np.arange(10)])
+def test_sct2e_wrapper(sclk_ticks):
+    """Test for `_sct2e_wrapper` function."""
+    et = _sct2e_wrapper(sclk_ticks)
+    if isinstance(sclk_ticks, float):
+        assert isinstance(et, float)
+    else:
+        assert len(et) == len(sclk_ticks)
```

Comment:
Can you also check that these equal the offset from the EPOCH, or does that not work due to leap seconds / other SPICE details?

```python
np.testing.assert_array_equal(et, EPOCH + sclk_ticks)
```

Reply:
That can be done for the test sclk kernel because the spacecraft clock is defined to tick at exactly one tick per second. Once we have real kernels, that is no longer true.

Reply:
Ahh, I see, that makes sense, thanks for clarifying. It might still be a better check here if we do have the known quantity in the tests so we are asserting that your TICK calculation was done correctly then I think, rather than just saying the returned arrays are the same shape.

Reply:
The TICK calculation only comes into play in the higher level `met_to_j2000ns` function and I do check that the value returned from that function matches the et calculated purely by SPICE... Now that I look at it, I am using the same `TICK_DURATION` so that test is a bit circular. I will change that test to calculate the tick duration using SPICE.